generate-release-notes.py

```diff
@@ -1,20 +1,39 @@
 #!/usr/bin/env python3
-from github import Github  # https://github.com/PyGithub/PyGithub
-import yaml
+# pylint: disable=missing-module-docstring
+# pylint: disable=missing-class-docstring
+# pylint: disable=missing-function-docstring
 import os
 import re
 import io
 import datetime
 
+# To install, run python3 -m pip install PyYAML
```

Comment:
We should be using a requirements.txt to do this, rather than having people install modules individaully. I certainly don't want to read the code in order to figure out how to prep my environment.

Ideally, this uses a virtualenv and/or a docker environment, and doesn't require me to even know I'm using Python.